cmd/templ/generatecmd/main.go

```diff
@@ -125,6 +125,38 @@ func runCmd(ctx context.Context, w io.Writer, args Arguments) (err error) {
 		p = proxy.New(args.ProxyPort, target)
 	}
 
+	if !args.KeepOrphanedFiles {
```

Comment:
I don't think this is the right place to do this work, because it means walking through the directory tree twice which is relatively expensive to do.

If we can combine the deletion of orphaned files with the generation, I think that would be better. Also, I'm wondering if the deletion of the orphaned file should be a warning or some other level to make it stand out.

Reply:
100% right, I was on something. Passed the argument to the processFile function (would that be the way?), checked it and used logWarning for the message.